web/themes/joinup/templates/navigation/menu-local-action.html.twig

```diff
@@ -10,11 +10,4 @@
  * @see template_preprocess_menu_local_action()
  */
 #}
-{%
-  set classes = [
-    'tab',
-    'tab--horizontal',
-    'mdl-tabs__tab',
-  ]
-%}
-<li{{ attributes.addClass(classes) }}>{{ link }}</li>
+<span{{ attributes }}>{{ link }}</span>
```

Comment:
Made a quick double-check on local action links when joinup theme is admin theme.
For this specific task, no problems found in content, people and structure-related management, so it's ok.

![content joinup](https://cloud.githubusercontent.com/assets/1923476/26361513/8870088c-3fdb-11e7-95ae-d8438a469679.png)